### ISSUE TITLE
AE issues

### ISSUE THREAD

**Issue by drdhaval2785** [sanskrit-lexicon/csl-devanagari#28]
https://github.com/sanskrit-lexicon/CORRECTIONS/issues/92#issuecomment-913174476 mentioned the following change to the data.

```
	Line 39391:	<div n="lb"/>{#????#}
		%%This line is extraneous; to be removed altogether.
```

Its effect has been given like the following.
```
<L>5145<pc>214<k1>imitate<k2>imitate
{@Imitate,@}¦ {%v. t.%} {#अनुकृ#} 8 U (oft. with gen.),
<div n="lb"/>{#अनुगम्-सृ#} 1 P, {#अनु-इ-या#} 2 P, {#अनुवृत्#} 1 A,
<div n="lb"/>{#अनुवि-धा#} 3 U, {#अनुवद्#} 1 P, {#अनुहृ#} 1 P,
<div n="lb"/>{#विडंब्#} 10. {@-ion,@} {%s.%} {#अनुकारः-करणं, अनु-#}
<div n="lb"/>{#-कृति#} {%f.,%} {#विडंबनं, अनुवृत्ति#} {%f.,%} {#अनुसरणं#}. {@2@}
<div n="lb"/>{#उपमानं, प्रतिरूपं, सादृश्यं#}. -{@ive,@} {%a.%} {#अनु-#}
<div n="lb"/>{#-करणशील, अनुकारिन्, अनुगामिन्#}. -{@or,@} {%s.%}
<div n="lb"/>{#अनुकर्तृ#} {%m.,%} {#अनुकारकः, अनुयायिन्-अनु-#}

<div n="lb"/>{#-कारिन्#} {%m.,%} {#अनुगतिकः#}.
<LEND>
```

I have not removed the line completely. Just put a blank line there.
I remember somewhere @funderburkjim was suggesting that we should refrain from changing line numbers, as some display code etc may be dependent on it. 
I am not sure whether it still remains the case.
Ideally we should remove this blank line, if it does not break something significant.

**Comment by drdhaval2785** [comment]
```
	Line 89525:	Purvapithi{??}a,
		Purvapithika, [properly, Pūrvapīṭhikā]
```
Kept Purvapithika only.
As Uttarapithika is also without diacritic.

**Comment by funderburkjim** [comment]
> I have not removed the line completely. Just put a blank line there.
I remember somewhere @funderburkjim was suggesting that we should refrain from changing line numbers,

In ae.txt,  the original digitization maintained the lines of the printed text.
And this provision has been maintained so far in ae.txt.   Many, but not all, of the dictionary
digitizations respect the printed text line breaks.

_Removing lines does NOT have an impact on any code.   Rather, the concern regards diverging from
the printed text._

Maintaining the lines causes many difficulties in analyzing.  The first most obvious problem is with
hyphenation caused by end of line in printed text.

In recent work with AP90,  I aimed to 'resolve' all the line breaks  and used the 'lbinfo' tag to provide
a trail back to the original lines.

This could be done with ae.txt.   But I am not sure the ap90 approach should be generally applied.  

An alternate view would be to simply drop the self-imposed requirement that digitizations originally
prepared with line-break observance maintain that line-break observance --- in other words, free
the digitization from the shackles of the rather arbitrary line breaks imposed by the printing process.

**Comment by drdhaval2785** [comment]
I feel that there are some positive sides of keeping line breaks. They are
1. The lines are human readable in majority of text editors and also on command line.
2. Texts which have relatively human readable sized lines are better amenable to git processes. If the line is too long, locating the actual change between two commits is a pain.
3. It makes physical comparision from printed text easier.

**Comment by funderburkjim** [comment]
> as some display code etc may be dependent on it.

Not display code.   But rather code used to generate and install changes.

For example, there is a utility (updateByLine.py)  that I use quite often to install changes in standard form.
The `standard` form for a change transaction is
```
LNUM old OLD TEXT OF LINE LNUM IN xxx.txt
LNUM new NEW TEXT
```

Note that the LNUM (line number in xxx.txt) is significant in this standard form.

python updateByLine.py xxx.txt changes.txt xxx_new.txt

This takes xxx.txt and a file (changes.txt) of change transactions and thereby generates a new version xxx_new.txt

Submitting changes in this manner is optimal.  

changes.txt can be generated either manually (for small lists)  or
by various kinds of programs.

**Comment by gasyoun** [comment]
> I feel that there are some positive sides of keeping line break

Agree. We can delete them only we have a place in code we could restore them if wanted again.
